lib/ws-kernel-picker.js

```diff
@@ -201,12 +194,12 @@ export default class WSKernelPicker {
     Session.startNew(sessionInfo.options).then(this.onSessionChosen.bind(this));
```

Comment:
Not really related to this PR, but what is ` Session.startNew(sessionInfo.options).then(this.onSessionChosen.bind(this));` supposed to do here?

Reply:
It starts a new session (as opposed to connecting to an existing one). `Session.startNew` returns a promise, and `this.onSessionChosen` accepts a session and uses it to create a WSKernel. `bind` helps set the `this` value correctly.

Reply:
Oh, ok thanks. So `startNew` returns a `session` and then `onSessionChosen` is called with that session.